common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1415,27 +1424,19 @@ object RssConf extends Logging {
       .checkValue(p => p >= 1024 && p < 65535, "invalid port")
       .createWithDefault(9096)
 
-  def metricsSystemEnable(conf: RssConf): Boolean = conf.get(METRICS_ENABLED)
-
-  def metricsSampleRate(conf: RssConf): Double = conf.get(METRICS_SAMPLE_RATE)
-
-  def metricsSamplePerfCritical(conf: RssConf): Boolean = {
-    conf.getBoolean("rss.metrics.system.sample.perf.critical", false)
-  }
-
-  def metricsSlidingWindowSize(conf: RssConf): Int = conf.get(METRICS_SLIDING_WINDOW_SIZE)
-
-  def innerMetricsSize(conf: RssConf): Int = {
-    conf.getInt("rss.inner.metrics.size", 4096)
-  }
-
-  def masterPrometheusMetricHost(conf: RssConf): String = conf.get(MASTER_PROMETHEUS_HOST)
-
-  def masterPrometheusMetricPort(conf: RssConf): Int = conf.get(MASTER_PROMETHEUS_PORT)
-
-  def workerPrometheusMetricHost(conf: RssConf): String = conf.get(WORKER_PROMETHEUS_HOST)
+  val METRICS_SAMPLE_PERFORMANCE_CRITICAL: ConfigEntry[Boolean] =
+    buildConf("celeborn.metrics.system.sample.perf.critical")
```

Comment:
would you please document it and add missing `version("0.2.0")` for those `ConfigEntry`s? we are not simply changing the `rss.` prefix to `celeborn.` but expect a meaningful and consistent renaming during migration